.gitignore

```diff
@@ -1 +1,4 @@
 config.py
+__pycache__/
+tutorial-env
```

Comment:
We usually name the venv folders as something like: `venv` or `ENV`: https://www.gitignore.io/api/python